nixos/modules/services/web-apps/shiori.nix

```diff
@@ -40,7 +46,7 @@ in {
       environment.SHIORI_DIR = "/var/lib/shiori";
 
       serviceConfig = {
-        ExecStart = "${package}/bin/shiori serve --address '${address}' --port '${toString port}'";
+        ExecStart = "${package}/bin/shiori serve --address '${address}' --port '${toString port}' --webroot '${webRoot}'";
```

Comment:
In the future, it might be a good idea to create an option named `extraOpts`, like the ones in the prometheus exporters, to be used as an escape hatch if other options like these are missing. But let's just go with this for now.